app/controllers/admin/events_controller.rb

```diff
@@ -15,5 +15,11 @@ class EventsController < Admin::ApplicationController
 
     # See https://administrate-docs.herokuapp.com/customizing_controller_actions
     # for more information
+
+    private
+
+    def resource_params
```

Comment:
Good catch. I created a `Admin::AdministrateCustomization` concern (and included it in those 3 controllers) and tested it in the admin interface and it works. Cool?